samples/testing/testify/suite_test.go

```diff
@@ -0,0 +1,60 @@
+package main
```

Comment:
It is a bit of a double up of https://pr90-dot-gotraining-testing.appspot.com/gocourse-2019-05-melbourne.slide#64 (Test two implementations of interface in suite) - but I guess that's acceptable.